handlers/handlers.go

```diff
@@ -128,9 +126,10 @@ func (i *Impl) GET_SensorData(rw http.ResponseWriter, req *http.Request, _ httpr
 	rw.Write(js)
 }
 
-func (i *Impl) GET_Faults(rw http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+func (i *Impl) GET_Faults_ID(rw http.ResponseWriter, req *http.Request, ps httprouter.Params) {
```

Comment:
Still working on this function and adding the `GET_FAULTS` back to only display the last message added